integration/tsconfig-test.ts

```diff
@@ -0,0 +1,136 @@
+import { test, expect } from "@playwright/test";
```

Comment:
Love this test. Do you think you could help me understand which bits and pieces are important in the assertions via something like this: https://kentcdodds.com/blog/aha-testing